markdown/ch2.markdown

```diff
@@ -296,7 +296,7 @@ By cutting the tree at a specific height, we can get a finer or coarser clusteri
 
 The `linkage` function in `scipy.cluster.hierarchy` performs a hierarchical clustering of the rows of a matrix, using a particular metric (for example, Euclidean distance, Manhattan distance, or others) and a particular linkage method, the distance between two clusters (for example, the average distance between all the observations in a pair of clusters).
 
-It returns the merge tree as a "linkage matrix", which contains each merge operation along with the distance computed for the merge and the number of observations in the resulting cluster. From the `linkage` documentation:
+It returns the merge tree as a "linkage matrix," which contains each merge operation along with the distance computed for the merge and the number of observations in the resulting cluster. From the `linkage` documentation:
```

Comment:
?

Reply:
As mentioned in ch1, yes, this is the correct English punctuation style, because writers were not trained in CS.